src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1551,6 +1551,39 @@ public void unpublishFeedbackSession(String feedbackSessionName,
             throw new RuntimeException(ERROR_SENDING_EMAILS, e);
         }
     }
+    
+    public EmailWrapper sendConfirmationEmailForSubmission(String courseId, String feedbackSessionName,
+                                                           String userId, String unregisteredStudentEmail,
+                                                           String regKey)
+                    throws EntityDoesNotExistException {
+        
+        FeedbackSessionAttributes session = getFeedbackSession(feedbackSessionName, courseId);
+        StudentAttributes student = null;
+        InstructorAttributes instructor = null;
+        
+        if (userId != null) {
+            student = studentsLogic.isStudentInAnyCourse(userId)
+                    ? studentsLogic.getStudentForCourseIdAndGoogleId(courseId, userId) : null;
+            instructor = instructorsLogic.isGoogleIdOfInstructorOfCourse(userId, courseId)
+                    ? instructorsLogic.getInstructorForGoogleId(courseId, userId) : null;
+        }
+        
+        if (student == null && unregisteredStudentEmail != null) {
+            student = new StudentAttributes();
+            student.email = unregisteredStudentEmail;
+            student.name = "";
```

Comment:
This means the email will read "Hello ,". Could be better if it reads "Hello teammatesuser@gmail.com,", i.e duplicate the email to the name field.